.buildkite/publish/dra/init_dra_publishing.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
```

Comment:
Are we allowed to run such scripts in Python?

Bash is sooo hard to read/write :/

Reply:
Yeah, we definitely could. I just copied this from prior DRA art for ent-search though. And I'd rather use a script that I know works already, than write a new one from scratch. 🙈